docs/10_domains/60_electrochemistry.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Electrochemistry"
+slug: "/electro_chemistry"
+---
+
+import { LbeChip } from "@site/src/components/lbe/LbeElements.js";
+import ElnFinder from "@site/src/components/eln/ElnFinder.js";
+
+<LbeChip title="physical chemistry" />
+<LbeChip title="electrochemistry" />
+
+## Introduction
+
+Electrochemistry plays a significant role in very different research areas, such as organic chemistry, analytical chemistry, life sciences, engineering and material science.
+Research is therefore often highly interdisciplinary, and the number of techniques and measurement procedures available are vast.
+
+## Data Types
+
+Conceptually, the main data type in electrochemistry consists of time series data containing fundamental electrical parameters such as voltage (potential)
+and current (current density), including the measurement time.
+For AC methods also the frequency, the impedance and the phase angle are relevant.
+These data are most simply stored as CSV, but are often only available in non-standard file formats.
+Often additional techniques are employed *operando* to gain more detailed insights into the origin of the measured charge transfer
+processes and other non-measurable (chemical) processes occurring concomitantly at the electrodes, in the electrolyte, or the interfaces,
+such as IR-spectroscopy, mass spectrometry or diffraction methods.
+This data is often stored along with the values from the electrochemical measurements.
```

Comment:
Gives the impression all data is in the same file.

Reply:
This is indeed sometimes the case.  I specify that both options are possible.